boards/arduino/portenta_h7/arduino_portenta_h7-common.dtsi

```diff
@@ -166,3 +168,43 @@ zephyr_udc0: &usbotg_fs {
 		};
 	};
 };
+
+&quadspi {
+	pinctrl-0 = < &quadspi_bk1_io0_pd11
+			&quadspi_bk1_io1_pd12
+			&quadspi_bk1_io2_pf7
+			&quadspi_bk1_io3_pd13
+			&quadspi_bk1_ncs_pg6
+			&quadspi_clk_pf10 >;
```

Comment:
This one is odd...

Note: You should use: tab = 8 spaces

Reply:
I was wondering why I couldn't get it. Thanks for the tip :)

Reply:
Should be done now.